cssinjs/index.ts

```diff
@@ -1,14 +1,15 @@
-import dedent from 'dedent'
 import { readFileSync } from 'fs'
+import { ensureFileSync } from 'fs-extra'
 import { install, updateFile } from 'mrm-core'
 
-module.exports = () => {
+module.exports = ({ isReact }: { isReact?: boolean } = {}) => {
```

Comment:
But what's the point? The whole module cssinjs is react specific and I can't imagine a way to use it in non react context